docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,45 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+Clone the main REANA repository:
+
+```console
+$ git clone https://github.com/reanahub/reana.git && cd reana
+```
```

Comment:
The cloning step is missing.  If we go for `hub`, we can automatise it even better :wink: so that people would onlly have to type `make`.